docs-src/stargate-develop/modules/develop/pages/tooling.adoc

```diff
@@ -168,23 +168,24 @@ The Swagger UI allows you to interact with the REST and Document APIs in a web b
 === Using the Swagger UI
 
 ifeval::["{product}" == "Astra DB"]
-In Astra, go to the Connect tab for your database, choose GraphQL under the
+In Astra, go to the Connect tab for your database, choose REST API or Document API under the
```

Comment:
I'm a bit confused on this part (and other Astra variants). It seems to mention REST API and Document API in the same context as the GraphQL playground? But the link is for swagger. So I guess it is the mention of the playground that should be replaced by swagger?

Reply:
I feel like something got overwritten. In my haste to fix it, I didn't read closely enough.

Reply:
I would NEVER leave sentences without periods at the end...

Reply:
Okay, should be correct now. Sorry about that.